i18n-cache/index.js

```diff
@@ -8,41 +8,56 @@ const fetch = require( 'node-fetch' );
 const supportedLocales = [
 	'ar', // Arabic
 	'bg', // Bulgarian
+	'bo', // Tibetan
+	'ca', // Catalan
 	'cs', // Czech
 	'cy', // Welsh
 	'da', // Danish
 	'de', // German
 	'en-au', // English (Australia)
 	'en-ca', // English (Canada)
 	'en-gb', // English (UK)
+	'en-nz', // English (New Zealand)
+	'en-za', // English (South Africa)
 	'el', // Greek
 	'es', // Spanish
+	'es-ar', // Spanish (Argentina)
+	'es-cl', // Spanish (Chile)
+	'es-cr', // Spanish (Costa Rica)
+	'fa', // Persian
 	'fr', // French
+	'gl', // Galician
 	'he', // Hebrew
 	'hr', // Croatian
 	'hu', // Hungarian
 	'id', // Indonesian
 	'is', // Icelandic
 	'it', // Italian
 	'ja', // Japanese
+	'ka', // Georgian
 	'ko', // Korean
 	'nb', // Norwegian (Bokmål)
 	'nl', // Dutch
+	'nl-be', // Dutch (Belgium)
 	'pl', // Polish
 	'pt', // Portuguese
 	'pt-br', // Portuguese (Brazil)
 	'ro', // Romainian
 	'ru', // Russian
 	'sk', // Slovak
 	'sq', // Albanian
+	'sr', // Serbian
 	'sv', // Swedish
 	'th', // Thai
 	'tr', // Turkish
+	'uk', // Ukrainian
+	'ur', // Urdu
+	'vi', // Vietnamese
 	'zh-cn', // Chinese (China)
 	'zh-tw', // Chinese (Taiwan)
 ];
 
-const getLanguageUrl = ( locale ) => `https://widgets.wp.com/languages/gutenberg/${ locale }.json`;
+const getLanguageUrl = ( locale ) => `https://translate.wordpress.org/projects/wp-plugins/gutenberg/dev/${ locale }/default/export-translations\?format\=json`;
```

Comment:
Do we have a good understanding of which code commit is the `dev` (`Development` on GlotPress) channel corresponding to? And in particular, how can we know that it perfectly matches the code version of gutenberg-mobile?

Reply:
Will go ahead and merge this PR. It'd still be useful if we can document the potential mismatch between our code version and the translations version on GlotPress.